.github/workflows/config.yml

```diff
@@ -7,9 +7,14 @@ on:
     types: [ synchronize, opened ]
   workflow_dispatch:
 
+env:
+  OPENSTUDIO_VER: "3.7.0-rc1"
+  OPENSTUDIO_SHA: "211bb633b0"
+  OPENSTUDIO_PLATFORM: "Darwin-x86_64"
```

Comment:
This is great. And we should use this regardless of whether we end of adopting `macos-latest` over `ubuntu-latest`.

Reply:
Using this [here](https://github.com/NREL/resstock/pull/1196).